deploy/kube-config/standalone-test/heapster-controller.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       labels:
         k8s-app: heapster-test
     spec:
+      serviceAccountName: heapster
       containers:
       - name: heapster-test
         image: k8s.gcr.io/heapster-amd64:v1.4.2
```

Comment:
We should probably bump the image version to v1.5.1

Reply:
Done.